src/test/kotlin/no/nav/syfo/melding/kafka/KafkaDialogmeldingFraBehandlerConsumerSpek.kt

```diff
@@ -100,6 +112,45 @@ class KafkaDialogmeldingFraBehandlerSpek : Spek({
                     pSvar.msgId shouldBeEqualTo msgId.toString()
                     pSvar.behandlerPersonIdent shouldBeEqualTo UserConstants.BEHANDLER_PERSONIDENT.value
                     pSvar.behandlerNavn shouldBeEqualTo UserConstants.BEHANDLER_NAVN
+                    val vedleggListe = database.getVedlegg(msgId)
+                    vedleggListe.size shouldBeEqualTo 0
+                }
+                it("Receive dialogmelding DIALOG_SVAR and known conversationRef and with vedlegg") {
+                    val dialogmeldingSendt = generateMeldingTilBehandlerRequestDTO().toMeldingTilBehandler(
+                        personident = UserConstants.ARBEIDSTAKER_PERSONIDENT,
+                    )
+                    database.connection.use {
+                        it.createMeldingTilBehandler(dialogmeldingSendt)
+                    }
```

Comment:
Mener det er en `createMeldingerTilBehandler`-testmetode for dette også.

Reply:
Fikser.